tgui/packages/tgui/interfaces/AntagInfoBrainwashed.tsx

```diff
@@ -57,7 +58,13 @@ const ObjectivePrintout = (props, context) => {
           objectives.map((objective) => (
             <>
               <Stack.Item key={objective.count}>
-                {objective.count}. {objective.explanation}
+                {objective.count}.{' '}
+                <span
+                  // eslint-disable-next-line react/no-danger
+                  dangerouslySetInnerHTML={{
+                    __html: sanitizeText(objective.explanation), // brainwashing objectives are sanitized anyways
+                  }}
```

Comment:
I dont like that players are allowed to use any HTML tags at all in their thing. This should be decoded on the byond side before being sent in as TGUI will sanitise it properly then

Reply:
It's sanitized on the BYOND side - this is due to the fact some objectives, like with cult, have `<b>` for example.

Reply:
Yea this is the correct way of doing this. Players can't write objective text most of the time and we already strip the input if so. sanitizeText adds extra safety by stripping scripts, links etc